conferences/abstracts/ukoswg_submission.md

```diff
@@ -0,0 +1,29 @@
+# UK-OSWG (UK Network of Open Science Working Groups)
+
+* Website: https://sites.google.com/view/ukoswg19
+* Application form: https://docs.google.com/forms/d/e/1FAIpQLSeUkvLx5WPTovNavwSPjZyvodqFXUNhHn8b0xUZmVHPJZdubA/closedform
+* Location: Aston University, Birmingham, UK
+* Date: 11 April 2019
+
+## Authors
+Patricia Herterich (University of Birmingham) on behalf of *The Turing Way Collaboration* and University of Birmingham Library Services
+
+## Session outline
+
+* Proposed (working) title:    
+The TuringWay - supporting reproducibility and Open Science from a librarian's perspective
+
+* Brief outline of session:  
+Professional services in research institutions such as librarians provide various services and guidance to support Open Science. 
+This session will introduce "The Turing Way" project, a 7 month project led by the Alan Turing Institute to bring together research software engineers and research data librarians in order to write a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is “too easy not to do”. 
+The talk will be starting point for a discussion how librarians in your institutions might already support Open Science and what you would want them to provide to make your Open Science Working Group more successful.
+
+* Type (e.g., workshop, talk, discussion group)  
+Input talk followed by discussion
```

Comment:
Is the word "input" supposed to be here?

Reply:
I think this is what I submitted but then there's only so much brain capacity that I put into this so looking at it now, it could have been phrased better.